dungeon/chain.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "dungeon",
+  "status": "live",
+  "network_type": "mainnet",
+  "chain_type": "cosmos",
+  "website": "https://www.cryptodungeon.org",
+  "pretty_name": "Dungeon Chain",
+  "chain_id": "dungeon-1",
+  "bech32_prefix": "dungeon",
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://rpc-dungeonchain.apeironnodes.com",
+        "provider": "ApeironNodes"
+      },
+      {
+        "address": "https://rpc-archive-dungeonchain.apeironnodes.com",
+        "provider": "ApeironNodes (Archive)"
+      }
+    ],
+    "rest": [
+      {
+        "address": "https://api-dungeonchain.apeironnodes.com",
+        "provider": "ApeironNodes"
+      },
+      {
+        "address": "https://api-archive-dungeonchain.apeironnodes.com",
+        "provider": "ApeironNodes (Archive)"
+      }
+    ]
+  },
+  "peers": {
+    "seeds": [],
+    "persistent_peers": [
+      {
+        "id": "node1",
```

Comment:
This should be node peer_id, not an arbitrary string.

Check the nodes via RPC using the `/status` endpoint to get the id.